api/src/extensions.ts

```diff
@@ -294,9 +294,20 @@ class ExtensionManager {
 	private initializeWatcher(): void {
 		logger.info('Watching extensions for changes...');
 
-		const localExtensionPaths = NESTED_EXTENSION_TYPES.flatMap((type) => {
-			const typeDir = path.posix.join(pathToRelativeUrl(env['EXTENSIONS_PATH']), pluralize(type));
-			const fileExts = ['js', 'mjs', 'cjs'];
+		const extensionsPath = pathToRelativeUrl(env['EXTENSIONS_PATH']);
+		const fileExts = ['js', 'mjs', 'cjs'];
+
+		const rootExtensionDirs = path.posix.join(extensionsPath, '{directus-extension-*,@directus/extension-*}');
```

Comment:
What's the idea behind `@directus/extension-*`?

Reply:
I used the `EXTENSION_NAME_REGEX` from `/packages/constants/src/extensions` to know what the expected structure for extension names was.  That regex includes the `@directus/extension-*` pattern. https://github.com/directus/directus/blob/main/packages/constants/src/extensions.ts#L29